contracts/src/TeleporterTokenSource.sol

```diff
@@ -205,12 +213,14 @@ abstract contract TeleporterTokenSource is
                 input.fallbackRecipient == address(0),
                 "TeleporterTokenSource: non-zero fallback recipient"
             );
-            adjustedAmount = _prepareSend(
-                input.destinationBlockchainID,
-                input.destinationBridgeAddress,
-                amount,
-                input.primaryFee
-            );
+            (adjustedAmount, feeAmount) = _prepareSend({
+                destinationBlockchainID: input.destinationBlockchainID,
+                destinationBridgeAddress: input.destinationBridgeAddress,
+                amount: amount,
+                feeTokenAddress: input.feeTokenAddress,
+                feeAmount: input.primaryFee,
+                isMultihop: isMultihop
```

Comment:
`isMultihop` is known to be false in all calls to `_prepareSend`. We should remove it as a parameter all together. 

`_prepareMultiHopRouting` was introduced to replace that case 👍